Dockerfile

```diff
@@ -5,6 +5,7 @@
 FROM node:18-bullseye as builder
 
 ARG MB_EDITION=oss
+ARG MB_VERSION
```

Comment:
Can we please use a custom name to avoid the confusion with [the official `MB_` prefixed variables](https://www.metabase.com/docs/latest/configuring-metabase/environment-variables)?

Reply:
done